Documentation/Storage-Configuration/Shared-Filesystem-CephFS/filesystem-storage.md

```diff
@@ -220,6 +220,168 @@ If the Rook cluster has more than one filesystem and the application pod is sche
+  * Modify the original name. To keep track, the best solution is to append to the original name the namespace name where you want your new PV. In this example `newnamespace`.
+  * Modify the volumeHandle. Again append the targeted namespace.
+  * Add the `staticVolume: "true"` entry to the volumeAttributes.
+  * Add the rootPath entry to the volumeAttributes, with the same content as `subvolumePath`.
+  * In the `nodeStageSecretRef` section, change the name to point to the secret you created earlier, `rook-csi-cephfs-node-user`.
+  * Remove the unnecessary information before applying the YAML (claimRef, managedFields,...):
+
+Your YAML should look like this:
+
+```yaml
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+  name: pvc-a02dd277-cb26-4c1e-9434-478ebc321e22-newnamespace
+spec:
+  capacity:
+    storage: 100Gi
+  csi:
+    driver: rook.cephfs.csi.ceph.com
+    volumeHandle: >-
+      0001-0011-rook-0000000000000001-8a528de0-e274-11ec-b069-0a580a800213-newnamespace
+    volumeAttributes:
+      clusterID: rook
+      fsName: rook-cephfilesystem
+      storage.kubernetes.io/csiProvisionerIdentity: 1654174264855-8081-rook.cephfs.csi.ceph.com
+      subvolumeName: csi-vol-8a528de0-e274-11ec-b069-0a580a800213
+      subvolumePath: >-
+        /volumes/csi/csi-vol-8a528de0-e274-11ec-b069-0a580a800213/da98fb83-fff3-485a-a0a9-57c227cb67ec
+      rootPath: >-
+        /volumes/csi/csi-vol-8a528de0-e274-11ec-b069-0a580a800213/da98fb83-fff3-485a-a0a9-57c227cb67ec
+      staticVolume: "true"
+    nodeStageSecretRef:
+      name: rook-csi-cephfs-node
+      namespace: rook
+    controllerExpandSecretRef:
+      name: rook-csi-cephfs-provisioner
+      namespace: rook
+  accessModes:
+    - ReadWriteMany
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: rook-cephfs
+  volumeMode: Filesystem
+```
+
+* In a new or other namespace, create a new PVC that will use this new PV you created. You simply have to point to it in the `volumeName` parameter. Make sure you enter the same size as the original PVC!:
+
+```yaml
+kind: PersistentVolumeClaim
+apiVersion: v1
+metadata:
+  name: second-pvc
+  namespace: newnamespace
+  finalizers:
+    - kubernetes.io/pvc-protection
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 100Gi
+  volumeName: pvc-a02dd277-cb26-4c1e-9434-478ebc321e22-newnamespace
+  storageClassName: rook-cephfs
+  volumeMode: Filesystem
+```
+
+You have now access to the same CephFS subvolume from different PVCs in different namespaces. Redo the previous steps (copy PV with a new name, create a PVC pointing to it) in each namespace you want to use this subvolume.
+
+### Shared volume removal
+
+As the same CephFS volume is used by different PVCs/PVs, you must proceed very orderly to remove it properly.
+
+* Delete the "cloned" PVCs in the different namespaces, but keep the original one!
```

Comment:
change `"cloned"` to static PVC's?  technically its not a cloned PVC

Reply:
Changed